### PR TITLE
Add rubocop as part of the rake command

### PR DESCRIPTION
**Pull request by jguitar**

### CODE REVIEW

**Comment by turniquette** [comment]
[PullReview stats](https://www.pullreview.com/github/TheTreeHouse/turniquette-api/reviews/pulls/pull_request_47)

**Comment by turniquette** [comment]
[PullReview](https://www.pullreview.com/github/TheTreeHouse/turniquette-api/reviews/pulls/pull_request_47?to_sha=036cd7344f22becf5f211d1fca662b8bb1ea604c) **[Great just fixed 7 issues,  19 to go.](https://www.pullreview.com/github/TheTreeHouse/turniquette-api/reviews/pulls/pull_request_47)**
[Test Infection](https://www.pullreview.com/github/TheTreeHouse/turniquette-api/reviews/pulls/pull_request_47) Your branch must be test infected.[ `app/controllers/application_controller.rb:1`](https://www.pullreview.com/github/TheTreeHouse/turniquette-api/reviews/pulls/pull_request_47)
[Test Infection](https://www.pullreview.com/github/TheTreeHouse/turniquette-api/reviews/pulls/pull_request_47) Your branch must be test infected.[ `test/models/event_test.rb:1`](https://www.pullreview.com/github/TheTreeHouse/turniquette-api/reviews/pulls/pull_request_47)
[Test Infection](https://www.pullreview.com/github/TheTreeHouse/turniquette-api/reviews/pulls/pull_request_47) Your branch must be test infected.[ `test/controllers/v1/registrations_controller_test.rb:1`](https://www.pullreview.com/github/TheTreeHouse/turniquette-api/reviews/pulls/pull_request_47)
[Test Infection](https://www.pullreview.com/github/TheTreeHouse/turniquette-api/reviews/pulls/pull_request_47) Your branch must be test infected.[ `test/controllers/v1/invitations_controller_test.rb:1`](https://www.pullreview.com/github/TheTreeHouse/turniquette-api/reviews/pulls/pull_request_47)
[Test Infection](https://www.pullreview.com/github/TheTreeHouse/turniquette-api/reviews/pulls/pull_request_47) Your branch must be test infected.[ `test/models/invitation_test.rb:1`](https://www.pullreview.com/github/TheTreeHouse/turniquette-api/reviews/pulls/pull_request_47)
and 2 others.

**Comment by scruti** [comment]
Wow, this is awesome (didn't know about the autocorrect).

**Comment by scruti** [comment]
Since this has 2 :+1: and I really want to have this feature in my workflow I merge it.
Thank you @jguitar !

File: Rakefile [hunk 1]
```diff
@@ -3,4 +3,12 @@
 
 require File.expand_path('../config/application', __FILE__)
 
+if Rails.env.development? || Rails.env.test?
+  require 'rubocop/rake_task'
+
+  RuboCop::RakeTask.new
+end
+
+task(:default).prerequisites << task(:rubocop)
```
Comment (gponsu) [inline Rakefile:12]: This line should not be in the upper block, inside if condition?

Comment (scruti) [inline Rakefile:12]: The default action for `rake` command is launch specs, and if you try to do it in production mode...

``` bash
rake RAILS_ENV=production
Abort testing: Your Rails environment is running in production mode!
```

**Pull request merged**